python-api/gstswitch/controller.py

```diff
@@ -295,8 +294,8 @@ def adjust_pip(self, xpos, ypos, width, height):
         try:
             conn = self.connection.adjust_pip(xpos, ypos, width, height)
             res = conn.unpack()[0]
-            print "adjust pip xpos:%s ypos:%s w:%s h:%s" % (
-                str(xpos), str(ypos), str(width), str(height))
+            print "adjust pip xpos:%u ypos:%u w:%u h:%u" % (
```

Comment:
I'm guessing these should actually be %i - but these prints should probably be removed anyway - see https://github.com/timvideos/gst-switch/issues/159